pages/store/[name]/index.tsx

```diff
@@ -11,22 +12,23 @@ const Store: React.FC<StoreProps> = ({ localstore }: any) => {
     const router = useRouter()
     if (router.isFallback) return <div>loading...</div>
     return (
-        <div className="flex flex-col">
+        <div className="flex flex-col bg-blue-100 ">
             <HeroSection
                 companyTitle={localstore.company}
                 companyDescription={localstore.description}
                 companyAddress={localstore.address}
                 companyTime={localstore.registered}
             />
             <div className="w-full">
-                <Categories
-                    productCategory={localstore.categories}
-                />
+               <ProductCard 
+               title={localstore.products}
+               description={localstore.productdescription}

```

Comment:
mejor usar camelCase, para las variables: 
```suggestion
               description={localStore.productDescription}

```